File: app/src/main/java/com/riders/thelab/data/remote/dto/artist/Artist.java
package com.riders.thelab.data.remote.dto.artist;

import com.squareup.moshi.Json;

import org.parceler.Parcel;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
@Parcel
public class Artist {
    @Json(name = "artistName")
    String artistName;
    @Json(name = "firstName")
    String firstName;
    @Json(name = "secondName")
    String secondName;
    @Json(name = "lastName")
    String lastName;
    @Json(name = "dateOfBirth")
    String dateOfBirth;
    @Json(name = "origin")
    String origin;
    @Json(name = "debutes")
    String debutes;
    @Json(name = "activities")
    String activities;
    @Json(name = "urlThumb")
    String urlThumb;
    @Json(name = "description")
    String description;
}

File: app/src/main/java/com/riders/thelab/data/remote/dto/artist/Artist.kt
package com.riders.thelab.data.remote.dto.artist

import kotlinx.serialization.SerialName
import java.io.Serializable

@kotlinx.serialization.Serializable
data class Artist constructor(
    @SerialName("artistName")
    var artistName: String,

    @SerialName("firstName")
    var firstName: String,

    @SerialName("secondName")
    var secondName: String,

    @SerialName("lastName")
    var lastName: String,

    @SerialName("dateOfBirth")
    var dateOfBirth: String,

    @SerialName("origin")
    var origin: String,

    @SerialName("debutes")
    var debutes: String,

    @SerialName("activities")
    var activities: String,

    @SerialName("urlThumb")
    var urlThumb: String,

    @SerialName("description")
    var description: String
): Serializable {
    var id: Int = 0
}